test/scenarios/reversible_heat_pump/inputs.json

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
I think the name "reversible" is a bit misleading here. But I can't think of another name either :D

Reply:
True! But I suppose reversible is still not wrong because while the model is not reversing, it models a real heat pump that, through hydraulics, does reverse heat flow.